main/src/modules/tools/opcPopulater/include/opcPopulater.h

```diff
@@ -19,6 +19,11 @@ class opcPopulater : public RFModule {
     double Y_ag;
     double Z_ag;
     double noise;
+    vector<double> spd1;
+    vector<double> spd2;
```

Comment:
I'd rather keep it, as it might be useful for future testing, as not everything in PPS-DAC integration is yet ready.